api/incubator/src/main/java/io/opentelemetry/api/incubator/config/internal/YamlStructuredConfigProperties.java

```diff
@@ -275,6 +283,20 @@ public String toString() {
     simpleEntries.forEach((key, value) -> joiner.add(key + "=" + value));
     listEntries.forEach((key, value) -> joiner.add(key + "=" + value));
     mapEntries.forEach((key, value) -> joiner.add(key + "=" + value));
+    nullKeys.forEach((key) -> joiner.add(key + "=null"));
     return joiner.toString();
   }
+
+  /**
+   * Returns a map representation of this {@link YamlStructuredConfigProperties}, which is useful
```

Comment:
nit: do we need to self-link the current type in javadoc? much shorter if we don't.